engine/src/main/java/org/destinationsol/game/sound/OggSoundManager.java

```diff
@@ -64,19 +110,38 @@ public OggSound getSound(String path, float basePitch) {
     }
 
     /**
-     * Plays a sound. Source must not be null.
+     * Plays a sound at specified position, or coming from specific source.
+     * <p>
+     * {@code source} must not be null if the sound is specified to loop, and at least one of {@code source} or
+     * {@code position} must be specified.
      *
-     * @param position         position of a sound. If null, source.getPosition() will be used
-     * @param source           bearer of a sound. Must not be null for looped sounds
-     * @param volumeMultiplier multiplier for sound volume
+     * @param game     Game to play the sound in.
+     * @param sound    The sound to play
+     * @param position Position to play the sound at. If null, source.getPosition() will be used.
+     * @param source   Bearer of a sound. Must not be null for looped sounds or when {@code position} is null.
+     */
```

Comment:
I find this javadoc redundant? It might be sufficient for overloaded methods like this to just reffer the other one with full javadoc.

Reply:
Yeah I might include just notes on what is unique in that method and that it then forwards to another method variant. But also does indeed not hurt - unless in the future one has its javadoc updated but not the other :-)

Tricky call.

Reply:
I'd say leave it in. By checking at  the libGdx code, I found out it's the same over there.